modules/Login/pages/Login.tsx

```diff
@@ -0,0 +1,41 @@
+import Button from '@modules/shared/components/atoms/Button/Button';
+import type { FC, ReactElement } from 'react';
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+import * as EButton from '../../shared/components/atoms/Button/types/EButton';
+import { loginUser, register } from '../../shared/api/auth';
+
+const Login: FC = (): ReactElement => {
+  const router = useRouter();
+  useEffect(() => {
+    if (document.cookie.split('=')[0] === 'user') {
+      router
+        .push('/')
+        .then()
+        .catch((err) => {
+          console.log(err);
+        });
+    }
+  }, []);
+
+  const login = async (): Promise<boolean> => {
+    loginUser();
+    const errorObj = await register();
```

Comment:
mmmm.. the fact that you named the variable errorObj indicates for me that you need to read more about promises and async operations.